sentry/templates/configmap-sentry.yaml

```diff
@@ -142,7 +142,7 @@ data:
     # and thus various UI optimizations should be enabled.
     SENTRY_SINGLE_ORGANIZATION = True
 
-    SENTRY_OPTIONS["system.event-retention-days"] = env('SENTRY_EVENT_RETENTION_DAYS') or 90
+    SENTRY_OPTIONS["system.event-retention-days"] = int(env('SENTRY_EVENT_RETENTION_DAYS') or 90)
```

Comment:
Not a python expect but should'nt `or 90` be outside the parenthesys?

Reply:
Nope, try it locally: 

```python
>>> number = int("") or 90
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: ''
>>> number = int("" or 90)
>>> print(number)
90
```

Reply:
Definitely I'm not a python expert ahah